gc/base/standard/ConcurrentGC.cpp

```diff
@@ -1874,7 +1874,7 @@ MM_ConcurrentGC::concurrentFinalCollection(MM_EnvironmentBase *env, MM_MemorySub
 		_concurrentPhaseStats._endTime = omrtime_hires_clock();
 		postConcurrentUpdateStatsAndReport(env);
 
-		if (env->acquireExclusiveVMAccessForGC(this, true, true)) {
+		if (acquireExclusiveVMAccessForGC(env, true)) {
```

Comment:
I missed in my original proposal that IU variant also has different behavior for start and end, so this did not end up as clean as I was hoping. 

I'll try again:

We could have 2 methods in base ConcurrentGC:
acquireExclusiveVMAccessFor(Cycle?)Start
acquireExclusiveVMAccessFor(Cycle?)End

The former would be virtual and specialize for IU and SATB. 
The later one would be in base ConcurrentGC, and no need to be virtual.

Neither would need any params (except Env, of course).